src/styles/gommette/components/_go-toggle.scss

```diff
@@ -0,0 +1,91 @@
+@import '../../utilities/helpers/bootstrap/variables';
```

Comment:
I think you must not import utilities variables because we already import https://github.com/Agorize/gommette/blob/dev/src/styles/gommette/helpers/_variables.scss

What do you think about this?

Reply:
yep, this is from when the style was in the component